polyfill/lib/ecmascript.mjs

```diff
@@ -5726,6 +5726,21 @@ export function ASCIILowercase(str) {
   ]);
 }
 
+export function ValueOfThrows(constructorName) {
```

Comment:
Isn't ecmascript.mjs intended to correspond with operations that are in current ECMA-262? It seems like the wrong home for this function.

Reply:
ecmascript.mjs is usually where we put all shared code. Unlike the spec files where there's mainadditions.html for the existing 262 overrides, in the polyfill things are all mixed together.  That said, this function isn't in the spec so I'll add a comment explaining why it exists.